src/cargo/util/command_prelude.rs

```diff
@@ -393,6 +393,28 @@ pub trait CommandExt: Sized {
         self._arg(unsupported_short_arg)
             ._arg(multi_opt("config", "KEY=VALUE", "Override a configuration value").global(true))
     }
+
+    fn arg_unstable_feature(self) -> Self {
```

Comment:
Why is this being moved to the prelude?  That is usually for when we need to share flag definitions but `-Z` is "global" meaning it being defined on the parent causes it to be inherited everywhere

Reply:
I want to put the real Arg and unsupported Arg together. 

Moved! Thanks!